apps/aecore/src/aec_fork_block_settings.erl

```diff
@@ -244,7 +244,20 @@ read_preset_contracts(?LIMA_PROTOCOL_VSN = Release) ->
     end.
 
 accounts_file_name(Release) ->
-    filename:join([dir(Release), accounts_json_file()]).
+    % check if: in devmode, a custom DB dir is defined & that dir cointains a custom accounts json (generated by devmode)
+    case aecore_env:is_dev_mode() and is_list(os:getenv("AE__CHAIN__DB_PATH")) andalso is_devmode_accounts_present() of
+        false ->
+            filename:join([dir(Release), accounts_json_file()]);
+        DevmodeAccsFilePath -> DevmodeAccsFilePath
+    end.
+    
+is_devmode_accounts_present() ->
+    Workspace = lists:last(filename:split(os:getenv("AE__CHAIN__DB_PATH"))),
+    FilePath = filename:join(os:getenv("AE__CHAIN__DB_PATH"), "devmode_prefunded_accs_" ++ Workspace ++ ".json"),
+    case filelib:is_regular(FilePath) of
+        true ->  FilePath;
+        false -> false        
+    end.
```

Comment:
I still think this is backwards, the node should not, I repeat not, have to know about the plugins, etc. What would be an ok solution here is to add a flag `YES_PLEASE_DO_GENERATE_ACCOUNTS` or `YES_PLEASE_USE_THESE_ACCOUNTS`. Where the first one is a boolean and the second one a path... 

Would that make it a lot more complicated?

Reply:
I can make devmode set an `AE_USE_CUSTOM_ACCOUNTS` env var and check for that 🤷‍♂️ ?